crawl-ref/source/webserver/server.py

```diff
@@ -232,6 +232,9 @@ def init_logging(logging_config):
         logging.getLogger().addFilter(TornadoFilter())
     logging.addLevelName(logging.DEBUG, "DEBG")
     logging.addLevelName(logging.WARNING, "WARN")
+    # Not a simple boolean check because the default (if unset/None) is True
+    if config.logging_config.get("access_logs") == False:
+        logging.getLogger("tornado.access").setLevel(logging.ERROR)
```

Comment:
trunk server.py `init_logging` already has some code that does this, should recognize the two. This was always supposed to be suppressed by default but until recently the suppression code only worked in tornado 2.